README.md

```diff
@@ -23,6 +23,28 @@ The arguments field on the New Action panel should be given the following:
 
 Also, the polling interval should match the value used in the "Repeat task every" field in the New Trigger dialog.  
 
+Step by step-
+How to install and configure Windows_Signal:
+  1. Install and configure Signal_iQ.
+  2. Download Windows_Signal repo contents and extract.
+  3. Copy report_event.py located in the Windows_Signal directory into the Signal_iQ\python\test\scripts directory.
+
+Test configuration:
+  1. Open Powershell
+  2. Change directories to the Windows_Signal directory.
+  3. Run this command>> C:\[path to Windows_Signal directory]\Send-Signal.ps1 -PyModule "C:\[path to Signal_iQ directory]\python\test\scripts\report_event.py" -EventsJsonFilePath "C:\[path to Windows_Signal directory]\json" -EnvironmentID [envID]
+  4. For more detailed output, use the -Verbose flag at the end of the previous step.
+       
+Create a scheduled task:
+  1. Open the Windows Task Scheduler
+  2. Create task
+  3. [General tab] Name- SIOS Signal; Security Options- Run whether user is logged on or not, Run with highest privileges
+  4. [Triggers tab] New trigger- Begin the task As startup, New trigger- On a schedule, Daily; advanced settings, repeat task every 5 minutes for a duration of indefinitely.
+  5. [Actions tab] New action- Action, start a program; Program/script, Powershell.exe; Add arguments, C:\[path to Windows_Signal directory]\Send-Signal.ps1 -PyModule "C:\[path to Signal_iQ directory]\python\test\scripts\report_event.py" -EventsJsonFilePath "C:\[path to Windows_Signal directory]\json" -EnvironmentID [envID]
```

Comment:
Wrap any code snippets in back ticks (\`) so that the readme formatting looks cleaner.